# Algorithm.wrapper

Public algorithms applied to LCFD analysis pipeline

#
### Broad Institute repository
https://github.com/broadinstitute

### DeepPrime
Deep learning-based model that can predict the prime editing efficiency for all possible pegRNAs for a given target sequence
http://deepcrispr.info/DeepPrime/

### Dana Pe'er
https://github.com/dpeerlab

#
# Single-cell platform

### 0. review
https://broadinstitute.github.io/2020_scWorkshop/

https://bioconductor.org/books/3.14/OSCA.advanced/

https://bioconductor.org/books/3.17/OSCA.basic/quality-control.html#quality-control-outlier  # 3MAD is described here.

https://hbctraining.github.io/scRNA-seq/schedule/

### 1. Seurat
https://satijalab.org/seurat/

https://github.com/satijalab

### 2. Annotation
SingleR:  (not official) https://biostatsquid.com/singler-tutorial/ , https://ruins880.tistory.com/106

sc-type:   https://github.com/IanevskiAleksandr/sc-type

Garnett:  https://cole-trapnell-lab.github.io/garnett/docs/

GPTCellType:  https://github.com/Winnie09/GPTCelltype

### 3. Alternative
scGPT:  https://github.com/bowang-lab/scGPT


#
# sc Trajectory analysis

### Overview
https://broadinstitute.github.io/2020_scWorkshop/trajectory-analysis.html

### monocle
monocle3: https://cole-trapnell-lab.github.io/monocle3

monocle2: https://github.com/cole-trapnell-lab

### velocity
Vignette1: https://velocyto.org/velocyto.py/tutorial/index.html # It starts from .bam/.sam files to yield a .loom file with counts divided in spliced/unspliced/ambiguous.

Vignette2: https://htmlpreview.github.io/?https://github.com/satijalab/seurat-wrappers/blob/master/docs/scvelo.html # We've moved on to python these days, so take this as a reference only.

Vignette3: https://scvelo.readthedocs.io/en/stable/ # to analyze velocity

### Palantir
https://palantir.readthedocs.io/en/latest/

### Waddington-OT
https://broadinstitute.github.io/wot/


#
# Cell-to-cell interaction

### CellChat
https://github.com/sqjin/CellChat

https://github.com/jinworks/CellChat?tab=readme-ov-file

Website: http://www.cellchat.org/

Vignette1: https://rdrr.io/github/sqjin/CellChat/f/tutorial/CellChat-vignette.Rmd

Vignette2: https://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/Comparison_analysis_of_multiple_datasets.html

### CellPhoneDB
https://github.com/Teichlab/cellphonedb
https://github.com/ventolab/CellphoneDB

visualization for CellPhoneDB result: https://github.com/martaint/InterCellar

### scTensor
https://github.com/rikenbit/scTensor

#
# ATAC analysis
Signac: https://stuartlab.org/signac/

Cicero: https://cole-trapnell-lab.github.io/cicero-release/docs_m3/

Cicero.Vignette1: https://stuartlab.org/signac/articles/cicero

Cicero.Vignette2: https://www.bioconductor.org/packages/devel/bioc/vignettes/cicero/inst/doc/website.html

ChromVar: https://greenleaflab.github.io/chromVAR/articles/Introduction.html

pycisTopic: https://github.com/aertslab/pycisTopic , https://pycistopic.readthedocs.io/en/latest/ # Must be done before SCENIC+

pycisTarget: https://github.com/aertslab/pycistarget , https://pycistarget.readthedocs.io/en/latest/ # Must be done before SCENIC+

#
# GRN analysis
pySCENIC: https://github.com/aertslab/pySCENIC?tab=readme-ov-file , https://pyscenic.readthedocs.io/en/latest/

SCENIC+: https://github.com/aertslab/scenicplus , https://scenicplus.readthedocs.io/en/latest/

#
# Genomics

### ChIPseeker
https://github.com/YuLab-SMU/ChIPseeker

### ClusterProfiler / Enrichment
https://yulab-smu.top/biomedical-knowledge-mining-book/index.html

https://ycl6.github.io/GO-Enrichment-Analysis-Demo/3_clusterProfiler.html#Load_data


### Greenleaf : chromVAR, motifmatchr, ArchR, ChrAccR
https://github.com/GreenleafLab




